### DISCUSSION TITLE
please help me  my movie collections

### DISCUSSION THREAD

**Discussion by haxhimitsu** [NAIST-SE/AttractingContributors#25] [General]
Please correct my movie lists!

https://github.com/haxhimitsu/AttractingContributors-moveis

**Comment by maxcell1024** [comment]
Added your movie list.
I'm a fan of Ghibli films.

Please let me know if there are any mistakes.

**Comment by yuka1369** [comment]
Added your movie list.

I add「Whiplash」your movie list. It blew me away.

I am beginner. so please tell me if I make a mistake.

**Reply by haxhimitsu** [reply]
Hello yuka1369.

Sorry, I can't find your pull request.

Could you check your repository status?
 
Thanks

**Comment by tattiyaS** [comment]
Add a Marvel movie to your list. Hope you enjoy!

**Reply by haxhimitsu** [reply]
Did you closed issue?
I can't merge your issue.
 Could you reopen yout issue please??

**Reply by tattiyaS** [reply]
ok sorry

**Reply by tattiyaS** [reply]
i have redo it, did you get it?

**Reply by haxhimitsu** [reply]
Thanks your efforts.

Compretely finished  your pull request.

I love MCU and Mad Max too!!

**Comment by vittunyuta-aom** [comment]
added!!

**Comment by mailtoy** [comment]
> Please correct my movie lists!
> 
> https://github.com/haxhimitsu/AttractingContributors-moveis

Thank you for your answer! 
And I added my answer to your readme. 👯‍♀️

**Comment by satokibi44** [comment]
added!

**Comment by manzoku23** [comment]
Added my favorite one!
Please help me 
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by Gloria109** [comment]
Thx, guys! Added to my fav list.
